connector/src/main/scala/com/google/cloud/spark/bigquery/BigQueryWriteHelper.scala

```diff
@@ -163,7 +170,7 @@ case class BigQueryWriteHelper(bigQuery: BigQuery,
     createTemporaryPathDeleter.deletePath
```

Comment:
Please add the condition from line 88, notice there's a bug in the current implementation, it should be `if (options.temporaryGcsBucket.nonEmpty)`

Reply:
Tks David. I made some changes to fix this. Can you take a look again?